src/providers/deno.rs

```diff
@@ -41,6 +53,18 @@ impl Provider for DenoProvider {
     }
 
     fn start(&self, app: &App, _env: &Environment) -> Result<Option<StartPhase>> {
+        // First check for a deno.json and see if we can rip the start command from there
+        if app.includes_file("deno.json") {
+            let deno_json: DenoJson = app.read_json("deno.json")?;
+
+            if let Some(tasks) = deno_json.tasks {
+                if let Some(start) = tasks.start {
```

Comment:
The Deno fresh start command uses `deno run` but to build we run [`deno compile`](https://deno.land/manual/tools/compiler) which "will compile the script into a self-contained executable". Does `deno run` use the compiled executable if it exists? Is `deno run` the recommended way of starting fresh in production?

Reply:
I wasn't sure, B U T, deno task start does seem to be the correct avenue for this

If there's a better way, we should update the examples. But this PR I think still stands cause we should be detecting the start command as `deno task start`, in the same way we detect `yarn start` etc